### PR TITLE
Add WaitServer

### PR DESCRIPTION
**Pull request by chasefleming**
Issue: #22

### CODE REVIEW

File: gateway/grpc.go [hunk 1]
```diff
@@ -206,6 +206,11 @@ func (g *GrpcGateway) Ping() error {
 	return g.client.Ping(ctx)
 }
 
+func (g *GrpcGateway) WaitServer() error {
```
Comment (ianthpun) [inline gateway/grpc.go:209]: do we want the context to come in as an arugment?

Comment (chasefleming) [reply gateway/grpc.go:209]: I was just keeping it the same pattern as `Ping`. Do you think it's necessary?

Comment (bjartek) [reply gateway/grpc.go:209]: Best practise is to always send in ctx ad first arg when doing network calls

Comment (bluesign) [reply gateway/grpc.go:209]: yeah I think we need to change all to have context in general. not sure about how we version here, but this is a breaking change.

Comment (chasefleming) [reply gateway/grpc.go:209]: Yeah if we want to add context to `Ping` we should do it on the stable cadence branch

**Review by ianthpun — APPROVED** [review]

**Pull request merged**